packages/store/src/operators/leaveNgxs.ts

```diff
@@ -0,0 +1,26 @@
+import { MonoTypeOperatorFunction, Observable, Observer } from 'rxjs';
+import { NgxsExecutionStrategy } from '../execution/symbols';
+
+/**
+ * Returns operator that will run
+ * `subscribe` outside of the ngxs execution context
+ */
+export function leaveNgxs<T>(
```

Comment:
could it be better to call it another way?
`ngxsStreamListener`

Reply:
Could you explain why?

Reply:
Because basically we do a wrapper over the behavior subject stream

Reply:
I think it should be named after what it acheives as opposed to how it does it.

Reply:
ok